api/src/test/java/org/openmrs/util/OpenmrsUtilTest.java

```diff
@@ -62,6 +71,14 @@
 	private static GlobalProperty luhnGP = new GlobalProperty(
 	        OpenmrsConstants.GLOBAL_PROPERTY_DEFAULT_PATIENT_IDENTIFIER_VALIDATOR,
 	        OpenmrsConstants.LUHN_IDENTIFIER_VALIDATOR);
+
+	private File createFolderInUserDirectory(String name) {
```

Comment:
Well according to what I have realized we are testing files in folders not folders in directories. Lets make this method to create a file given it's relative path name eg. 

```java
String relativePath = "{folderName}" + seperator + "{fileName}"; // The relative path could be the folder name + the file we wanna test
```

That way we can make code more reusable

Reply:
ok